src/Resources/config/backup.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016-2018 Spomky-Labs
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+use BehatExtension\DoctrineDataFixturesExtension\Service\Backup\BackupInterface;
+use BehatExtension\DoctrineDataFixturesExtension\Service\Backup\MysqlDumpBackup;
+use BehatExtension\DoctrineDataFixturesExtension\Service\Backup\PostgresqlDumpBackup;
+use BehatExtension\DoctrineDataFixturesExtension\Service\Backup\SqliteCopyBackup;
+use BehatExtension\DoctrineDataFixturesExtension\Service\BackupService;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return function (ContainerConfigurator $container) {
```

Comment:
Fluent configurators appeared only in symfony 3.4

https://symfony.com/blog/new-in-symfony-3-4-php-based-configuration-for-services-and-routes

While behat declares DI dependency as "~2.1||~3.0||~4.0"

https://github.com/Behat/Behat/blob/master/composer.json#L23

So we either should raise the minimal DI requirement to `3.4` ot downgrade the configuraton format to some classics